packages/neos-ui/src/Containers/LeftSideBar/NodeTree/Node/index.js

```diff
@@ -131,7 +131,7 @@ export default class Node extends PureComponent {
                     isHidden={$get('properties._hidden', node)}
                     isHiddenInIndex={$get('properties._hiddenInIndex', node)}
                     hasError={this.hasError()}
-                    label={$get('label', node)}
+                    label={$get('label', node).replace(/<\/?[^>]+(>|$)/g, '')}
```

Comment:
I think not, as it's not a dev dep. It's enough to just put it as a dep to something that uses it, in this case it's neos-ui.
@grebaldi might say if I'm wrong.